packages/web/src/components/gcds-textarea/gcds-textarea.stories.tsx

```diff
@@ -63,41 +63,41 @@ export default {
   },
 };
 
-const Template = (args) => `
+const Template = (args) => (`
 // Web Component (Angular, Vue)
 <gcds-textarea
   textarea-id="${args.textareaId}"
   label="${args.label}"
-  hint="${args.hint}"
-  error-message="${args.errorMessage}"
-  required="${args.required}"
-  disabled="${args.disabled}"
-  rows="${args.rows}"
-  value="${args.value}"
-  hide-label="${args.hideLabel}"
-  lang="${args.lang}"
+  ${args.hint ? `hint="${args.hint}"` : null}
+  ${args.errorMessage ? `error-message="${args.errorMessage}"` : null}
```

Comment:
Same comment about the error message: would you mind setting a default data type?